dapr/aio/clients/grpc/client.py

```diff
@@ -1103,6 +1108,260 @@ async def unlock(
         return UnlockResponse(status=UnlockResponseStatus(response.status),
                               headers=await call.initial_metadata())
 
+    async def start_workflow(
+            self,
+            workflow_component: str,
+            workflow_name: str,
+            input: Optional[bytes],
+            instance_id: Optional[str] = "",
+            workflow_options: Optional[Dict[str, str]] = dict()) -> StartWorkflowResponse:
+        """Starts a workflow.
+
+            Args:
+                workflow_component (str): the name of the workflow component
+                                    that will run the workflow. e.g. `dapr`.
+                workflow_name (str): the name of the workflow that will be executed.
+                input (Optional[bytes]): the input that the workflow will receive.
+                instance_id (Optional[str]): the name of the workflow instance,
+                                    e.g. `order_processing_workflow-103784`.
+                workflow_options (Optional[Dict[str, str]]): the key-value options
+                                    that the workflow will receive.
+
+            Returns:
+                :class:`StartWorkflowResponse`: Instance ID associated with the started workflow
+        """
+        # Warnings and input validation
+        warn('The Workflow API is an Alpha version and is subject to change.',
+             UserWarning, stacklevel=2)
+        validateNotBlankString(instance_id=instance_id,
+                               workflow_component=workflow_component,
+                               workflow_name=workflow_name)
+        encoded_data = json.dumps(input).encode("UTF-8")
+        # Actual start workflow invocation
+        req = api_v1.StartWorkflowRequest(
+            instance_id=instance_id,
+            workflow_component=workflow_component,
+            workflow_name=workflow_name,
+            options=workflow_options,
+            input=encoded_data)
+
+        try:
+            response, _ = self._stub.StartWorkflowAlpha1.with_call(req)
```

Comment:
Why are we using `with_call`? Why can't we just do `self._stub.StartWorkflowAlpha1(req)`? I think the point of my previous comment on this was missed.